internal/statestore/backfill.go

```diff
@@ -0,0 +1,152 @@
+package statestore
```

Comment:
Splitting the files up seems ok to me (redis.go is getting quite big, this would only make it bigger), however the existing redis.go has more than just tickets in it.  It also has generic redis connection stuff.  

Please keep common redis stuff to redis.go, splitting out tickets stuff to tickets.go (preferred), or add everything to redis.go directly.

I would prefer that this split/movement/cleanup was done in a separate PR, though I won't require it for approval this time.

Reply:
I like the idea of leaving 3 files - redis, backfill, ticket. Let's do that right now, anyway I've already started doing that.